src/test/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetailsTest.java

```diff
@@ -142,4 +142,52 @@ public void tesValidateQuestionDetails() {
         assertEquals(FeedbackNumericalScaleQuestionDetails.NUMSCALE_ERROR_MIN_MAX, errors.get(0));
         assertEquals(FeedbackNumericalScaleQuestionDetails.NUMSCALE_ERROR_STEP, errors.get(1));
     }
+
+    @Test
+    public void testShouldChangesRequireResponseDeletion_differentMinScale_shouldReturnTrue() {
+        FeedbackNumericalScaleQuestionDetails numDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMinScale(1);
+
+        FeedbackNumericalScaleQuestionDetails newNumDetails = new FeedbackNumericalScaleQuestionDetails();
+        numDetails.setMinScale(3);
```

Comment:
This should set min scale for `newNumDetails`. 
```suggestion
        newNumDetails.setMinScale(3);
```